index.html

```diff
@@ -4,9 +4,44 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:wght@100;300;400&family=Roboto:wght@100;300;400&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400&family=Roboto:wght@100;300;400&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Mobile version skeleton</title>
 </head>
 <body>
+    <header class="header">
+        <h1 class="nav_bar_text">My Logo</h1>
+        <nav class="app_bar_container">
+            <img class="burger_menu" src="Images\Normal Button\Tertiary\Enabled.png" alt="burger menu">
+        </nav>
+    </header>
     
+    <section class="headline">
+        <div class="main_text_container">
+            <h1 class="main_text">Hey There. I'm Lewis.</h1>
+        </div>
+        <div class="subject_text_container">
+            <h3 class="subject_text">I'm a Software Developer</h3>
+        </div>
+        <div class="description_text_container">
+            <p class="description_text">I can help you build a product, feature or website Look through 
+            some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        </div>
+        <div class="social_media">
+            <div id="github_icon"><img src="Microverse icons\Github Icon\Tertiary\Disabled.png" alt="Github link"></div>
+            <div id="linkedin_icon"><img src="Microverse icons\LinkedIn Icon\Tertiary\Disabled.png" alt="LinkedIn link"></div>
+            <div id="peace_icon"><img src="Microverse icons\Peace Symbol\Tertiary\Enabled.png" alt="Peace link"></div>
+            <div id="twitter_icon"><img src="Microverse icons\Twitter Icon\Tertiary\Disabled.png" alt="Twitter link"></div>
+            <div id="M_icon"><img src="Microverse icons\M Symbol\Tertiary\Disabled.png" alt="M-symbol link"></div>
+        </div>
```

Comment:
- [ ] Please 🙏🏻  consider using the `ul` tag as a container and put all the social media icon links in individual `li` tags, it will make your code more professional and semantically correct. It will also help **visually-impaired** people to navigate through your website with a good *User Experience* and please also use `a` tag to wrap your image icon so that people can connect with you on different platforms.

- [ ] Please once you change it make sure that you must have to add styles for it in the CSS and use the flexbox there to align the item properly.

- [ ] Please also fix your path name for the image, as you can see in the below image your icons image is broken. Kindly follow the github convention for giving the path for any assets file. Your Design should looks similar to the original design.

![image](https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/66476329/ad2d2fd1-c175-4d17-9d56-554333fdc337)


- [OPTIONAL] Please make sure that always close your tag with the right `tagName` otherwise as your code scale or as you will add more and more features to it, later on, it will be difficult for you to find the error just because of a *small* mistake.